tagsets/calendar.php

```diff
@@ -89,6 +89,10 @@ function calendar__get_events($admin = false, $start_time = 0, $end_time = 0, $a
         $query.=" AND ".table('experiments').".experimenter LIKE :admin_id ";
         $pars[':admin_id']='%|".$admin_id."|%';
```

Comment:
Though, you did not touch this line in the commit, it looks like single brackets needs to be used before/after $admin_id concatenation? Correct me if I am wrong.